src/main/java/org/folio/consortia/repository/criteria/CQL2JPACriteria.java

```diff
@@ -0,0 +1,82 @@
+package org.folio.consortia.repository.criteria;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Path;
+import jakarta.persistence.criteria.Predicate;
+import jakarta.persistence.criteria.Root;
+import lombok.extern.log4j.Log4j2;
+import org.folio.cql2pgjson.exception.CQLFeatureUnsupportedException;
+import org.folio.cql2pgjson.exception.QueryValidationException;
+import org.folio.cql2pgjson.util.Cql2SqlUtil;
+import org.z3950.zing.cql.CQLNode;
+import org.z3950.zing.cql.CQLParseException;
+import org.z3950.zing.cql.CQLParser;
+import org.z3950.zing.cql.CQLTermNode;
+
+import java.io.IOException;
+
+@Log4j2
+public class CQL2JPACriteria<E> {
```

Comment:
I think we don't need CQL2JPACriteria, at least for now

Reply:
updated